cmd/soroban-cli/src/config/secret.rs

```diff
@@ -55,13 +62,22 @@ pub struct Args {
     pub secure_store: bool,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum Secret {
-    SecretKey { secret_key: String },
-    SeedPhrase { seed_phrase: String },
+    SecretKey {
+        secret_key: String,
+    },
+    SeedPhrase {
+        seed_phrase: String,
+    },
     Ledger,
-    SecureStore { entry_name: String },
+    SecureStore {
+        entry_name: String,
+        #[serde(skip)]
+        #[serde(default)]
+        cached_entry: Arc<OnceLock<SecureStoreEntry>>,
```

Comment:
Here is where we are caching the specific SecureStoreEntry to reuse, so we don't get prompted multiple times per command